drivers/wifi/winc1500/Kconfig.winc1500

```diff
@@ -15,6 +15,11 @@ config WIFI_WINC1500_NAME
 	string "Driver name"
 	default "WINC1500"
 
+config WINC1500_DRV_USE_OLD_SPI_SW
+	bool "Driver to use Old SPI software"
```

Comment:
Please replace the description with "Use old SPI access method in the vendor HAL". And the help text: "This option, when enabled, causes USE_OLD_SPI_SW setting to be passed to vendor HAL".

Reply:
@pfalcon Do you want the issue description to be - "Use old SPI access method in the vendor HAL"??

Reply:
> Do you want the issue description to be - "Use old SPI access method in the vendor HAL"??

No, that wouldn't properly describe the nature of the patch.  As you can fit only so much information into a single line of <70 chars, better let it stay like it is.

Reply:
@pfalcon, I thought so too. Then what exactly do you mean by "Please replace the description with "Use old SPI access method in the vendor HAL".  Please exlain.

I understand what you wanted me to do with the help text.

Reply:
> Then what exactly do you mean by "Please replace the description with "Use old SPI access method in the vendor HAL". Please explain.

Currently the line in your patch to which this comment is attached reads:

~~~
	bool "Driver to use Old SPI software"
~~~

instead, it's suggested to be:
~~~
	bool "Use old SPI access method in the vendor HAL"
~~~

Reply:
That really makes sense! Thanks.

Reply:
OK, done. Please check. I will wait for the HAL pull request to go through, so that I can use the commit ID to make a final commit here after I squash them all in one commit.